### PR TITLE
[plugin.video.hdhomerun.simple] 1.0.0

### PR DESCRIPTION
**Pull request by Lunatixz**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by Lunatixz** [comment]
requires:
https://github.com/xbmc/repo-scripts/pull/822
https://github.com/xbmc/repo-scripts/pull/823

File: plugin.video.hdhomerun.simple/resources/lib/hdhomerun.py [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+    def __init__(self):
+        self.cache  = SimpleCache()
+        self.pyHDHR = PyHDHR.PyHDHR()
+        self.tuners = self.getTuners()
+        self.hasDVR = self.pyHDHR.hasSDDVR()
+        
+        
+    def browseTuners(self):
+        if PTVL_RUNNING: self.browseTunerMenu('All')
+        elif len(self.tuners) == 1: self.browseTunerMenu(self.tuners[tunerkey].getLocalIP())
```
[the remark this answers is not in the document]
Comment (Lunatixz) [reply plugin.video.hdhomerun.simple/resources/lib/hdhomerun.py]: False issue :)

File: plugin.video.hdhomerun.simple/addon.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        <import addon="script.module.uepg" version="1.0.3"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.service" library="service.py" start="login"/>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Simple HDHomeRun Interface.</summary>
+        <description lang="en_GB">Automatically find all the compatible HDHomeRun devices on your network, Simple and Easy HDHomeRun Interface.</description>
+        <disclaimer lang="en">Requires a compatible HDHomeRun device</disclaimer>
```
Comment (enen92) [inline plugin.video.hdhomerun.simple/addon.xml]: en_gb

Comment (Lunatixz) [reply plugin.video.hdhomerun.simple/addon.xml]: @enen92 fixed :)

**Pull request merged**